### PR TITLE
Editorial: list changes since last publication

### PR DESCRIPTION
**Pull request by marcoscaceres**
In preparation for final CR.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/pull/835.html" title="Last updated on Feb 18, 2019, 3:08 PM UTC (454d901)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/835/19b9590...454d901.html" title="Last updated on Feb 18, 2019, 3:08 PM UTC (454d901)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,6 +119,55 @@ <h1 id="title">
         There has been no change in dependencies on other workings groups
         during the development of this specification.
       </p>
+      <section>
+        <h3>
+          Changes since last publication
+        </h3>
+        <p>
+          Substantive changes to Payment Request API since the 9 July 2018
+          version are as follows. If complete list of changes, including all
```
Comment (ianbjacobs) [inline index.html]:
```suggestion
          version are as follows. The complete list of changes, including all
```

File: index.html [hunk 2]
```diff
@@ -119,6 +119,55 @@ <h1 id="title">
...
+          editorial changes is viewable in the <a href=
```
Comment (ianbjacobs) [inline index.html]:
```suggestion
          editorial changes, is viewable in the <a href=
```

File: index.html [hunk 3]
```diff
@@ -119,6 +119,55 @@ <h1 id="title">
...
+          "https://github.com/w3c/payment-request/commits/gh-pages">commit
+          history</a>.
+        </p>
+        <ul>
+          <li>Added support for notification when the user selects a payment
+          method, but before confirming payment. This allows merchant to update
```
Comment (ianbjacobs) [inline index.html]:
```suggestion
          handler, but before confirming payment. This allows merchant to update
```

File: index.html [hunk 4]
```diff
@@ -119,6 +119,55 @@ <h1 id="title">
...
+          <li>Removed <code>languageCode</code> and <code>regionCode</code>
+          from <a>PaymentAddress</a>.
+          </li>
+          <li>Removed <code>currencySystem</code>.
+          </li>
+          <li>"<a>Payment request is showing</a>" boolean now attached to
+          top-level browsing context. Previously, only a single payment UI was
+          allowed to be shown at a time across the whole browser. This now
+          allows multiple browser tabs to show a payment UI at the same time
+          (for payment handlers that can support that).
```
Comment (ianbjacobs) [inline index.html]:
```suggestion
          (for those payment handlers able to support it).
```

**Comment by marcoscaceres** [comment]
@ianbjacobs, don't forget to ✅ or else I can't merge 🙏

**Review by ianbjacobs — APPROVED** [review]

**Pull request merged**